src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -1,79 +1,244 @@
 package org.broadinstitute.dsde.workbench.leonardo.dao
 
 import com.google.api.services.dataproc.model._
-import org.broadinstitute.dsde.workbench.leonardo.config.DataprocConfig
+import com.google.api.services.dataproc.model.{Cluster => GoogleCluster}
```

Comment:
Minor, maybe `{Cluster => DataprocCluster}`

Also I feel like an "Optimize Imports" would be useful here.

Reply:
I made it GoogleCluster so that we could differentiate Google's Cluster from our Cluster, so I thought GoogleCluster was clearer  - but we could probably use a tie breaker here! a 

@helgridly ?

Reply:
Agreed on the imports though, I'll clean it up

Reply:
I'm fine with `GoogleCluster`. It disambiguates whose it is, which is the ambiguous thing. We know it's a dataproc cluster really.